packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/components/LogoModalStepper/tests/ImageCardAsset.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { render as renderTL } from '@testing-library/react';
+import { ThemeProvider, lightTheme } from '@strapi/design-system';
+
+import ImageCardAsset from '../ImageCardAsset';
+
+const ASSET_FIXTURES = {
+  ext: '.jpeg',
+  height: 250,
+  name: 'asset.jpeg',
+  size: 46.26,
+  url: 'uploads/asset.jpeg',
+  width: 340,
+};
+
+const render = (props) =>
+  renderTL(
+    <ThemeProvider theme={lightTheme}>
+      <IntlProvider locale="en" messages={{}} textComponent="span">
+        <ImageCardAsset asset={ASSET_FIXTURES} {...props} />
+      </IntlProvider>
+    </ThemeProvider>
+  );
+
+describe('ApplicationInfosPage | ImageCardAsset', () => {
+  it('should render and match snapshot', () => {
+    const { container } = render();
+
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should display and format asset extension and size', () => {
+    const { getByText } = render();
+
+    expect(getByText('.JPEG - 340✕250')).toBeInTheDocument();
```

Comment:
Can we make this two checks?
```suggestion
    expect(getByText('.JPEG')).toBeInTheDocument();
    expect(getByText('340✕250')).toBeInTheDocument();
```

This easily can break and separating both ensures we test on an atomic level ...

Reply:
make sense, not sure how to handle it though, if I test 
```js
`expect(getByText('.JPEG')).toBeInTheDocument();`
```
I get this error: `Unable to find an element with the text: .JPEG. This could be because the text is broken up by multiple elements.` Because it cannot break down the full sentence I guess?

I also tried with a regex but then I get multiple results matching (see DOM snapshot)

<img width="567" alt="image" src="https://user-images.githubusercontent.com/71838159/205650462-965d3977-ebb6-411a-92aa-ab98e44fedc1.png">

Reply:
could do `getAllByText` and ensure they're both in there?